volmaster/volmaster/cli.go

```diff
@@ -12,6 +12,9 @@ import (
 	"github.com/codegangsta/cli"
 )
 
+// version is provided by build
+var version = ""
```

Comment:
Well, we'll be removing the volmaster after release and rewriting it as 
an API service only, so that will be the time to consider it.

-Erik

On 23 May 2016, at 15:53, Madhav Puri wrote:

>> @@ -12,6 +12,9 @@ import (
>>  	"github.com/codegangsta/cli"
>>  )
>>
>> +// version is provided by build
>> +var version = ""
>
> @erikh , this PR will bring in the equivalent of  `docker --version`, 
> which is much smaller and local in scope but is useful for tying our 
> binaries to respective releases.
>
>  `docker version` and `netctl version` have a larger and distributed 
> scope of having a REST endpoint to deal with various distributed 
> components of a system. We should perhaps debate it's need in a 
> separate issue.
>
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/volplugin/pull/275/files/c911929805c12cdf0d07fe82e19281b680e8c08d#r64303226